docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -11,47 +11,34 @@ Introduction
 
 When writing applications for the ledger in Java, you want to work with a representation of Daml templates and data types in Java that closely resemble the original Daml code while still being as true to the native types in Java as possible. To achieve this, you can use Daml to Java code generator ("Java codegen") to generate Java types based on a Daml model. You can then use these types in your Java code when reading information from and sending data to the ledger.
 
-Download
-========
-
-You can download the `latest version <https://search.maven.org/artifact/com.daml/codegen-java>`__  of the Java codegen. Make sure that the following versions are aligned:
-
-* the downloaded Java codegen jar file, eg. x.y.z
-* the dependency to :ref:`bindings-java <daml-codegen-java-compiling>`, eg. x.y.z
-* the ``sdk-version`` attribute in the :ref:`daml.yaml <daml-yaml-configuration>` file, eg. x.y.z
-
 .. _daml-codegen-java-running:
 
 Run the Java codegen
 ====================
 
-The Java codegen takes Daml archive (DAR) files as input and generates Java files for Daml templates, records, and variants. For information on creating DAR files see :ref:`assistant-manual-building-dars`. To use the Java codegen, run this command in a terminal:
-
-.. code-block:: none
-
-  java -jar <path-to-codegen-jar>
-
-Use this command to display the help text:
+The Java codegen can be run from within your Daml project using the Daml assistant. Use this command to display the help text:
 
 .. code-block:: none
 
-  java -jar codegen.jar --help
+  daml codegen java --help
```

Comment:
maybe also worth showing how you can configure it in daml.yaml but doesn’t have to be in this PR.

Reply:
Oh, I didn't know you could, thanks!

Reply:
https://github.com/digital-asset/daml/issues/10183